src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -7028,7 +7028,7 @@ WorldSafeLocsEntry const* ObjectMgr::GetClosestGraveyard(float x, float y, float
     {
         if (z > -500)
         {
-            TC_LOG_ERROR("misc", "ZoneId not found for map {} coords ({}, {}, {})", MapId, x, y, z);
+            TC_LOG_ERROR("misc", "ZoneId not found for map {} coords ({}, {}, {}), player GUID: {}", MapId, x, y, z, playerGuid.GetRawValue());
```

Comment:
Do not call any functions when logging guids, ObjectGuid has a fmt::formatter specialization so just use `playerGuid`